static/app/utils/profiling/hooks/useAggregateFlamegraphQuery.ts

```diff
@@ -20,6 +20,9 @@ export function useAggregateFlamegraphQuery({transaction}: {transaction: string}
     return conditions.formatString();
   }, [transaction]);
 
+  const enabled =
+    !!transaction && Array.isArray(selection.projects) && selection.projects.length > 0;
```

Comment:
Is this to not fire the query before the selection is ready? This does assume that the flamegraph query cant be cross-project which is probably a reasonable assumption.

Reply:
yeah, correct. I think we always end up with a single project here, so I think its fine too